qiskit_aer/backends/aer_simulator.py

```diff
@@ -318,6 +318,12 @@ class AerSimulator(AerBackend):
     * ``accept_distributed_results`` (bool): This option enables storing
       results independently in each process (Default: None).
 
+    * ``runtime_parameter_bind_enable`` (bool): If this option is True
+      parameters are binded at runtime by using multi-shots without constructing
```

Comment:
`binded` -> `bound` because other descriptions use the latter in Aer.

Reply:
fixed